test/test-values-tiny.yaml

```diff
@@ -18,20 +18,37 @@ _000_commonConfig:
 
 factoryinsight:
   replicas: 1
+
+redpanda:
+  resources:
+    cpu:
+      cores: 1
+    memory:
+      enable_memory_locking: true
+      container:
+        max: 3Gi
+  console:
+    enabled: false
```

Comment:
Can't we enable it here, and remove our dependency on standalone console ?

Reply:
It's already like that in the default values. The `test/test-values-tiny.yaml` values are just for the CI, to test that everything works while having the minimum services possible. Since no one would access the console from the GH runner, there is no point to enable it. 
In the `test/test-values-full.yaml` values is enabled, so it's also tested in the CI